src/runtime.js

```diff
@@ -22,18 +22,20 @@ socket.events.on('offline', () => {
 module.exports.start = options => {
   options = options || {};
   config.host = options.host || config.host;
-  if (options.uuid && options.secret) {
-    credentials.setDeviceCredentials(options.uuid, options.secret);
+  if ((options.uuid || options.deviceUuid) && options.secret) {
+    credentials.setDeviceCredentials(options.uuid || options.deviceUuid, options.secret);
+  } else if (options.networkUuid && options.apiKey) {
+    credentials.setNetworkCredentials(options.networkUuid, options.apiKey);
```

Comment:
The main reason I didn't use the word "secret" was because we can't really consider it to be a secret once it's added into a mobile app. Hopefully we can reuse this value as an apiKey later, also my expectation is that this apiKey is mostly for read only API calls and event bus messages, once we add authorization.